styles.css

```diff
@@ -380,6 +384,74 @@ li {
   background: #606bc3;
 }
 
+.modal_footer {
+  display: flex;
+  flex-direction: column;
+}
+
+.section_modal {
+  display: none;
+}
+
+.show {
+  z-index: 10;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  inset: 0;
+  backdrop-filter: blur(5px);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.modal {
+  background-color: white;
+  position: relative;
+  width: 90%;
+  height: 90%;
+  max-width: 720px;
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: hidden;
+  margin: 15px;
+}
+
+.modal_content {
+  padding: 5%;
+}
+
+.header_modal {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.title_modal {
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.modal_buttons {
+  display: flex;
+  justify-content: center;
+  gap: 5%;
+}
+
+.button_modal {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+}
+
+.image_modal {
+  width: 100%;
+  max-width: 350px;
+}
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design.

  - The image width should fix the modal as it is in the Figma design.
  - You have blurred the modal container background but the blur color effect does not match the Figma design as you can see below :point_down: 

  ### Your design
  ![Screenshot from 2023-04-11 21-51-34](https://user-images.githubusercontent.com/92303891/231298664-494a44df-6528-4642-ac97-0fced952672e.png)
  
  ### Figma design
  ![Screenshot from 2023-04-11 21-47-21](https://user-images.githubusercontent.com/92303891/231298933-3dd6f394-bd9d-4094-9078-bec7588cf817.png)